### PR TITLE
Reimplementation of card based on dobi and ilyushin

### PR DESCRIPTION
**Pull request by tom-parker**

### CODE REVIEW

**Comment by tom-parker** [comment]
@matthew-andrews @keirog

File: bower.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "main.js"
   ],
   "dependencies": {
-    "next-buttons": "^1.0.0",
-    "o-expander": "^1.0.0",
+    "next-buttons": "https://github.com/Financial-Times/next-buttons.git#^v1.0.0",
     "dom-delegate": "^2.0.3",
-    "next-sass-setup": "^6.0.0"
+    "next-sass-setup": "^6.0.0",
+    "o-date": "~1.1.1"
```
Comment (keirog) [inline bower.json]: No tildes please!

**Comment by keirog** [comment]
:+1:

File: templates/partials/subheading.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+{{#unless hideSubheading}}
```
Comment (matthew-andrews) [inline templates/partials/subheading.html]: is it a good time to request that this be changed to be `showSubheading` ?

**Pull request merged**